pyproject.toml

```diff
@@ -65,27 +65,33 @@ source = ["src"]
 omit = ["*/.tox/*", "*/__main__.py", "*/setup.py", "*/.venv*/*", "*/venv*/*"]
 
 [tool.coverage.report]
-fail_under = 77
 show_missing = true
 skip_covered = false
 
 [tool.tox]
 legacy_tox_ini = '''
 [tox]
-envlist = py38, py39
+envlist = py{38,39}-{integration,unit},
```

Comment:
I think that we can simplify this more.  There is no need to really create separate env for the different Python versions.  `tox` is smart enough when running `tox -e py38-integration` and combined with filter under commands it will pick Python 3.8 is installed on the machine.

But I'm not sure how much more time I should use on optimizing this.  For now, I think this works good enough.